tests/spec_decode/e2e/test_ngram_correctness.py

```diff
@@ -211,3 +211,50 @@ def test_ngram_disable_queue(baseline_llm_generator, test_llm_generator,
                                          batch_size,
                                          max_output_len=output_len,
                                          force_output_len=True)
+
+
+@pytest.mark.parametrize(
+    "common_llm_kwargs",
+    [{
+        "model": "JackFram/llama-68m",
+
+        # Skip cuda graph recording for fast test.
+        "enforce_eager": True,
+
+        # Required for spec decode.
+        "use_v2_block_manager": True
+    }])
+@pytest.mark.parametrize("per_test_common_llm_kwargs", [{}])
+@pytest.mark.parametrize("baseline_llm_kwargs", [{}])
+@pytest.mark.parametrize(
+    "test_llm_kwargs",
+    [
+        {
+            "speculative_model": "[ngram]",
+            "num_speculative_tokens": k,
+            "ngram_prompt_lookup_max": 3,
+            "spec_decoding_acceptance_method": "typical_acceptance_sampler"
+        }
+        # Try a range of common k, as well as large speculation.
+        for k in [1, 3, 5]
+    ])
+@pytest.mark.parametrize(
+    "output_len",
+    [
+        # Use smaller output len for fast test.
+        32,
+    ])
+@pytest.mark.parametrize("seed", [1])
+@pytest.mark.parametrize("batch_size", [1, 32])
+def test_ngram_typical_acceptance_sampling(baseline_llm_generator,
```

Comment:
I think we don't need this test since you already have coverage of the framework using the draft model. There's no expected difference with ngram vs other proposal methods.

Can you check how much time this test adds? if it's >1min we should remove

Reply:
In its current form it was adding about 1.5 minutes. I removed this e2e test for now. We can add it back if needed later.